CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this extension will be documented in this file.
 
 ## Unreleased
 
+### Changed
+
+- Schema upload actions are now available in the Schemas view, as well as any Schema Registry shown
+  in the Resources view. Schemas can also now be uploaded from an editor without needing to save to
+  a file first.
+- Clicking a schema item in the Topics/Schemas view will now open the schema definition in a
+  read-only document without needing to click on the "View Schema" action.
+
```

Comment:
> Suggest an additional line stating that schema evolution is done with two gestures on the schema/subject element: one to download the latest version into a writeable buffer, and then a followup using the 'cloud upload' icon to submit any buffer or file's contents back up to the schema registry.

This is the same as what we had before though, isn't it? The cloud upload icon was just at the file editor nav area instead of in the sidebar view(s).

Reply:
But agreed, having a better icon for evolving a schema would be preferred. 👍

Reply:
> The cloud upload icon was just at the file editor nav area instead of in the sidebar view(s).

Yeah, but 'cause it was attached to the buffer, and not the subject or schema registry rows, it was somewhat "more clear" about which action does what. Now that they're both on the same row, even though they're currently subtly ordered correctly (click on the 'versions' icon first, then click on the upload icon when done editing), is inherently more confusing.

I think am generally saying that we probably need an extension walkthrough section on schema editing when all is said and done, 'cause it just isn't out of the box brain dead simple (nor was it ever before).